BasicTypesProxy.cxx

```diff
@@ -151,13 +236,11 @@ namespace caf
     const T val = GetValue();
 
     if constexpr(std::is_floating_point_v<T>){
-      if(isnan(val) || isinf(val)){
-        std::cout << "SRProxy: Warning: " << fName << " = " << val;
-        if(fTree && fTree->GetDirectory() && fTree->GetDirectory()->GetFile()){
-          std::cout << " in entry " << fEntry << " of " << fTree->GetDirectory()->GetFile()->GetName();
-        }
-        std::cout << std::endl;
-      }
+      std::string filename = (fTree && fTree->GetDirectory() && fTree->GetDirectory()->GetFile()) ? fTree->GetDirectory()->GetFile()->GetName() : "";
+      if(isnan(val))
```

Comment:
That's actually already handled in `BasicTypesProxy.h`:
https://github.com/cafana/SRProxy/blob/master/BasicTypesProxy.h#L454-L469

Reply:
I had looked for that, and missed it.
However, I really strongly dislike having scope global `using` directives in a header file, and I believe this is how the overriding should be done.
Second however: this is not your concern in this PR, and fixing it might have ripples in user code. So I bite it.